lib/game.rb

```diff
@@ -1,26 +1,19 @@
 require_relative 'opponent_factory'
 require_relative 'grid_factory'
+require_relative 'marks'
 
 class Game
 
-  def initialize
-    @grid_factory = GridFactory.new
-  end
-
   def players_and_marks(first_player, ui)
     {:first_player => {:player => first_player,
-                       :mark => "X"},
+                      :mark => Marks::X},
     :second_player => {:player => opponent(ui),
-                       :mark => "O"}
+                      :mark => Marks::O}
```

Comment:
`0` is that the letter O or the number zerro?
`I` is that an i upper case of low case L?

some upper case char can be tricky to recognise, it can be easier to use `cross` and `round for the marks

Reply:
I get it. But where did you see the I?

Reply:
It was just another example :)